monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -207,9 +207,7 @@ groups:
   - name: pools
     rules:
       - alert: pool full
-        expr: |
-          ceph_pool_stored / ceph_pool_max_avail
-          * on(pool_id) group_right ceph_pool_metadata * 100 > 90
+        expr: ceph_pool_stored / (ceph_pool_stored + ceph_pool_max_avail) * on(pool_id) group_right ceph_pool_metadata > 0.9
```

Comment:
Without the multiplication by hundred and an appropriate comparison (`* 100 > 90`) the description of the resulting alert will not be correct.

Reply:
@p-se Sorry I didin't realized thata this change was superseded by https://github.com/ceph/ceph/pull/32892
I'll close this PR.

Reply:
Oh right, good catch. In master this changed was over written by fb51c589b5b5. @p-se want me to open a tracker ticket for that?

Reply:
@alfonsomthd Sorry, it seems I added to confusion! I don't think it was superseded but accidentally removed. This PR is still valid IMO. Just in case we want to also backport https://github.com/ceph/ceph/commit/fb51c589b5b5cd6a05cbfb08c41ca46a8941b269, we will also need to backport the [fix that accidentally removed a valid correction](https://github.com/ceph/ceph/pull/33655).

@jan--f I created a tracker ticket and PR https://github.com/ceph/ceph/pull/33655 already, but thank you!